docker-compose.demo.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.2'
 services:
   api:
-    image: kqueen/api:v0.19
+    image: kqueen/api:v1.0
```

Comment:
I would not do this as part of the PR, just do the `setup.py` change. You need to merge this PR and tag the merge commit as `v1.0` so the image can get built, it won't exist until you push the tag, so I would create separate PR with the demo file update after the images are built already and you can test them